src/components/HeroHeader/DesktopHero.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image';
+import React from 'react';
+import Button from '../Button/Button';
+import Navbar from '../Navbar';
+import bgImage from '../../../public/images/hero-image.svg';
+
+export default function HeroHeader() {
+  return (
+    <div>
+      <Image
+        src={bgImage}
+        alt=""
+        objectFit="cover"
+        layout="fill"
+        className="-z-10 fixed top-0 opacity-[0.85]"
+        priority
+      />
+      <div className="relative">
+        <Navbar />
+      </div>
+      <div className="w-full flex flex-col items-center justify-center z-10">
+        <h1 className="font-montserrat font-bold text-center text-white text-[40px] leading-10 w-[388px] mx-auto mt-20">
+          React Developer Community Kenya
+        </h1>
+        <p className="font-montserrat text-white w-[570px] mx-auto text-center mt-5 mb-[45px]">
+          We are a tech community in Kenya focused on the React JS library and
+          its ecosystem.
+        </p>
+        <div className="flex items-center justify-center mb-44 relative">
+          <a
+            target="_blank"
+            href="https://bit.ly/joinreactdevske"
+            rel="noopener noreferrer"
+          >
+            <Button className="bg-[#EC0505] w-[307px] h-[61px] rounded-md text-white font-montserrat font-bold">
+              Join ReactDevsKe
+            </Button>
+          </a>
```

Comment:
Wrapping the button inside the link creates two nested tabbable elements, we should get rid of the `a` and use the `LinkButton` instead of the normal `Button`.

Reply:
Fixed via [102c9e6](https://github.com/reactdeveloperske/reactdevske-website/pull/105/commits/102c9e6d96fd1bdbe13da1a8e6343a327a357a77)